web/src/js/components/ContentView/ContentViewOptions.jsx

```diff
@@ -28,5 +29,8 @@ export default connect(
     state => ({
         contentViewDescription: state.ui.flow.viewDescription,
         readonly: !state.ui.flow.modifiedFlow,
-    })
+    }),
+    {
+        uploadContent: uploadContent
```

Comment:
Are you sure about this change? I think this gets passed in here: https://github.com/mitmproxy/mitmproxy/blob/5f91b7a45d0b983d3e0f3fff1be8af93736d35c6/web/src/js/components/FlowView/Messages.jsx#L114